### PR TITLE
Visual rework

### PR DESCRIPTION
**Pull request by Ctakayama**
This has all the new fonts, recolors, UI updates, etc. This is the finalized design we decided on last week as realized as possible. Team tab has been removed as well to make room for the meteor

### CODE REVIEW

File: root/src/js/github/issues.js [hunk 1]
```diff
@@ -56,19 +58,20 @@ function getGithubIssues(token) {
 
                         inputElement.addEventListener('change', function() {
                             if (this.checked) {
-                                var response = confirm(
```
Comment (vermar0187) [inline root/src/js/github/issues.js:59]: I believe we still want to be able to confirm closing the issue.

**Comment by vermar0187** [comment]
Looks good to me, just one small comment

File: root/src/js/github/issues.js (hunk 1 above)
Comment (Ctakayama) [reply root/src/js/github/issues.js:59]: for security reasons they don't let u style confirm dialogues, so if we keep them they're gonna look kind of botched like how they currently are

**Comment by Ctakayama** [comment]
need assistance refactoring to get code climate to agree with the code

**Pull request merged**